[2022-10-01T21:34:03.778+0000] {taskinstance.py:1165} INFO - Dependencies all met for <TaskInstance: rede_mercado.colector scheduled__2022-09-01T00:00:00+00:00 [queued]>
[2022-10-01T21:34:03.791+0000] {taskinstance.py:1165} INFO - Dependencies all met for <TaskInstance: rede_mercado.colector scheduled__2022-09-01T00:00:00+00:00 [queued]>
[2022-10-01T21:34:03.791+0000] {taskinstance.py:1362} INFO - 
--------------------------------------------------------------------------------
[2022-10-01T21:34:03.792+0000] {taskinstance.py:1363} INFO - Starting attempt 1 of 1
[2022-10-01T21:34:03.792+0000] {taskinstance.py:1364} INFO - 
--------------------------------------------------------------------------------
[2022-10-01T21:34:03.812+0000] {taskinstance.py:1383} INFO - Executing <Task(PythonOperator): colector> on 2022-09-01 00:00:00+00:00
[2022-10-01T21:34:03.818+0000] {standard_task_runner.py:54} INFO - Started process 82 to run task
[2022-10-01T21:34:03.820+0000] {standard_task_runner.py:82} INFO - Running: ['***', 'tasks', 'run', 'rede_mercado', 'colector', 'scheduled__2022-09-01T00:00:00+00:00', '--job-id', '352', '--raw', '--subdir', 'DAGS_FOLDER/dag.py', '--cfg-path', '/tmp/tmpmey1l6fw']
[2022-10-01T21:34:03.821+0000] {standard_task_runner.py:83} INFO - Job 352: Subtask colector
[2022-10-01T21:34:03.822+0000] {dagbag.py:525} INFO - Filling up the DagBag from /opt/***/dags/dag.py
[2022-10-01T21:34:04.393+0000] {task_command.py:384} INFO - Running <TaskInstance: rede_mercado.colector scheduled__2022-09-01T00:00:00+00:00 [running]> on host dc20523c063c
[2022-10-01T21:34:04.487+0000] {taskinstance.py:1592} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=rede_mercado
AIRFLOW_CTX_TASK_ID=colector
AIRFLOW_CTX_EXECUTION_DATE=2022-09-01T00:00:00+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2022-09-01T00:00:00+00:00
[2022-10-01T21:34:04.489+0000] {tasks.py:39} INFO - Defining DEC to export information
[2022-10-01T21:34:04.496+0000] {tasks.py:343} INFO - Reading prep dataframe
[2022-10-01T21:34:04.513+0000] {tasks.py:146} INFO - Instantiating the Browser with the requested settings
[2022-10-01T21:34:04.514+0000] {taskinstance.py:1851} ERROR - Task failed with exception
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/operators/python.py", line 175, in execute
    return_value = self.execute_callable()
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/operators/python.py", line 193, in execute_callable
    return self.python_callable(*self.op_args, **self.op_kwargs)
  File "/opt/airflow/dags/tasks.py", line 345, in colector
    browser = config_browser(config)
  File "/opt/airflow/dags/tasks.py", line 149, in config_browser
    desired_capabilities=config["browser_arg"])
  File "/home/airflow/.local/lib/python3.7/site-packages/selenium/webdriver/remote/webdriver.py", line 252, in __init__
    raise WebDriverException("Desired Capabilities must be a dictionary")
selenium.common.exceptions.WebDriverException: Message: Desired Capabilities must be a dictionary

[2022-10-01T21:34:04.528+0000] {taskinstance.py:1406} INFO - Marking task as FAILED. dag_id=rede_mercado, task_id=colector, execution_date=20220901T000000, start_date=20221001T213403, end_date=20221001T213404
[2022-10-01T21:34:04.541+0000] {standard_task_runner.py:107} ERROR - Failed to execute job 352 for task colector (Message: Desired Capabilities must be a dictionary
; 82)
[2022-10-01T21:34:04.595+0000] {local_task_job.py:164} INFO - Task exited with return code 1
[2022-10-01T21:34:04.629+0000] {local_task_job.py:273} INFO - 0 downstream tasks scheduled from follow-on schedule check
